components/item/AwesomeSwiper.vue

```diff
@@ -99,6 +104,9 @@
       },
       updateSwiper() {
         this.swiper.update();
+      },
+      isSinglePlayableMedia() {
```

Comment:
Move to computed properties, replacing `playableMedia`.

Reply:
ok